shopping-cart.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The Sandwich Collective Shopping Cart</title>
+    <link rel="stylesheet" href="src/main.css">
+    <link rel="stylesheet" href="src/cart.css">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Tajawal:400,700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>
+            The Sandwich Collective
+        </h1>
+        <img src="assets/shop-image.png" alt="small shop image" class="logo-image">
+    </header>
+
+    <main>
+        <h2>Your Cart</h2>
+        <table>
+            <thead>
+                <tr>
+                    <th>Item</th>
+                    <th>Quantity</th>
+                    <th>Item Price</th>
+                    <th>Total</th>
+                </tr>
+            </thead>
+            <tbody id="cart-content">
+            </tbody>
+            <tfoot>
+                <tr>
+                    <th></th>
+                    <th></th>
+                    <th>Order Total:</th>
+                    <th id="table-footer"></th>
+                </tr>
+            </tfoot>
+        </table>
+    
+        <section id="button-section">
+            <p>Enter a promo code: 
+                <input type="text" id="promo-code"> <button value="apply" id="apply-promo-code">Apply</button>
+            </p>
+            <p>
+                <button value="order">Order Now</button>
+            </p>
+        </section>
+    </main>
+    <script src="/src/register.js" type="module"></script>
```

Comment:
you don't need a script tag for register.js because those functions are imported in and running through shopping-cart.js